#!/usr/bin/env python
# -*- coding: utf-8 -*-

import math
from dynamic_reconfigure.parameter_generator_catkin import (
    int_t,str_t, ParameterGenerator)

"""
.. codeauthor:: Olaya Alvarez Tunon
: file Winch.cfg
"""

gen = ParameterGenerator()

# Add available winch options.
gen.add("winch_port_baudrate", int_t, 0, "transmitting baudrate for port",115200,9600,115200)
gen.add("winch_port",          str_t, 0, "name of port for altimeter",    "/dev/ttyUSB1")


exit(gen.generate("stams_gui", "depthstamp", "Winch"))
